internal/local/check.go

```diff
@@ -98,8 +98,8 @@ type State struct {
 
 // Checker defines the shape a checker must have in order to be executed as vulcan-check.
 type Checker interface {
-	Run(ctx context.Context, target string, opts string, state astate.State) error
-	CleanUp(ctx context.Context, target string, opts string)
+	Run(ctx context.Context, target, targetType, opts string, state astate.State) error
+	CleanUp(ctx context.Context, target, targetType, opts string)
```

Comment:
s/targetType/assetType/g

Reply:
Oh, seems like I mishandled the renaming, at least for the cleaning method :(
Will fix all these! Thx! I must be blind!